src/red_commons/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Common utilities used by multiple projects maintained by Cog Creators."""
+from . import logging
 
 __version__ = "1.0.0"
-
-__all__ = ()
+__all__ = ("logging",)
```

Comment:
I don't want module names to be re-exported here.
```suggestion

__all__ = ()
```